docs/plus/03-jenkins.md

```diff
@@ -14,7 +14,7 @@ You need the following tools installed on your Jenkins CI server:
 * Node
 * Karma
 
-Optional we highly suggest to install the following Jenkins plug-in:
+Following Jenkins plugin is optional, but next guidelines are based on it:
```

Comment:
Probably better: `The following Jenkins plugin, but the next guidelines are based on it:`